src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -152,36 +153,38 @@ class MLParams(
 
   /** Returns a copy of the current MLParams instance */
   def copy(
-      // *** Common to all algorithms ***
-      randomSeed: Option[Int] = randomSeed,
-      numExamples: Option[Long] = numExamples,
-      numTestExamples: Option[Long] = numTestExamples,
-      numPartitions: Option[Int] = numPartitions,
-      // *** Specialized and sorted by name ***
-      bucketizerNumBuckets: Option[Int] = bucketizerNumBuckets,
-      depth: Option[Int] = depth,
-      docLength: Option[Int] = docLength,
-      elasticNetParam: Option[Double] = elasticNetParam,
-      family: Option[String] = family,
-      featureArity: Option[Int] = featureArity,
-      k: Option[Int] = k,
-      link: Option[String] = link,
-      maxIter: Option[Int] = maxIter,
-      numClasses: Option[Int] = numClasses,
-      numFeatures: Option[Int] = numFeatures,
-      numInputCols: Option[Int] = numInputCols,
-      numItems: Option[Int] = numItems,
-      numUsers: Option[Int] = numUsers,
-      vocabSize: Option[Int] = vocabSize,
-      optimizer: Option[String] = optimizer,
-      regParam: Option[Double] = regParam,
-      rank: Option[Int] = rank,
-      smoothing: Option[Double] = smoothing,
-      tol: Option[Double] = tol): MLParams = {
+    // *** Common to all algorithms ***
```

Comment:
The old indentation here was correct; it's the indentation change above in the constructor which needs to be reverted.